### PR TITLE
refactor: make seata-config more readable

### PR DESCRIPTION
**Pull request by koonchen**
### Ⅰ. Describe what this PR did

I found some optimization points when i was reading source code in seata-config module.

### CODE REVIEW

File: config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
     private static final int MAX_CONFIG_OPERATE_THREAD = 2;
 
-    private static final long LISTENER_CONFIG_INTERNAL = 1 * 1000;
+    private static final long LISTENER_CONFIG_INTERNAL = 1000;
```
Comment (zjinlei) [inline config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java]: 1 * 1000 is more readable

File: config/seata-config-etcd3/src/main/java/io/seata/config/etcd3/EtcdConfiguration.java [hunk 2]
```diff
@@ -240,7 +238,7 @@ private static Client getClient() {
     /**
      * the type config change notifier
      */
-    private class EtcdListener implements ConfigurationChangeListener {
+    private static class EtcdListener implements ConfigurationChangeListener {
```
Comment (zjinlei) [inline config/seata-config-etcd3/src/main/java/io/seata/config/etcd3/EtcdConfiguration.java]: Why use static?

File: config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java (hunk 1 above)
Comment (koonchen) [reply config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java]: ok

File: config/seata-config-etcd3/src/main/java/io/seata/config/etcd3/EtcdConfiguration.java (hunk 2 above)
Comment (koonchen) [reply config/seata-config-etcd3/src/main/java/io/seata/config/etcd3/EtcdConfiguration.java]: EtcdListener/NacosListener/ZKListener does not have access to the enclosing instance. It uses less space.

**Pull request merged**